safegraph/params.json.template

```diff
@@ -1,17 +1,23 @@
 {
-  "static_file_dir": "./static",
-  "raw_data_dir": "/mnt/data/safegraph/",
-  "export_dir": "./receiving",
-  "cache_dir": "./cache",
-  "n_core": "12",
-  "aws_access_key_id": "",
-  "aws_secret_access_key": "",
-  "aws_default_region": "",
-  "aws_endpoint": "",
-  "sync": true,
-  "wip_signal" : ["median_home_dwell_time_7dav",
-                  "completely_home_prop_7dav",
-                  "part_time_work_prop_7dav",
-                  "full_time_work_prop_7dav"],
-  "log_exceptions": false
+  "common": {
```

Comment:
in #795 , you use `shared` instead of `common`, is that change intentional? 

Also, is export dir  here isntead of `indicator` because it's also shared with the archive step?

Reply:
No they should both be the same.  I just changed #795 to be `common` although I can switch them all to `shared` instead too.

Export dir is a parameter shared in the validation step too.

Reply:
oh right, because you need to validate the output :facepalm:. `common` sounds good to me.